app/js/daemonManager.js

```diff
@@ -72,17 +72,21 @@ function DaemonManager() {
 		// duration that it takes to load up the blockchain.
 		apiCall("/daemon/updates/check", function(err, update) {
 			if (err) {
+				console.log(err, update)
 				self.Running = false;
 				// Check again later
-				setTimeout(updatePrompt, 1000);
+				setTimeout(updatePrompt, 5000);
```

Comment:
I definitely don't think that you should be flipping `Daemon.Running` based on the outcome from a call to /daemon/updates/check`. The only api call that should be really used to check if the daemon is working is `/daemon/version`